lang/csharp/src/apache/main/File/DataFileConstants.cs

```diff
@@ -21,9 +21,6 @@ namespace Avro.File
     /// <summary>
     /// Constants used in data files.
     /// </summary>
-    [System.Diagnostics.CodeAnalysis.SuppressMessage("Design",
-        "CA1052:Static holder types should be Static or NotInheritable",
-        Justification = "Maintain public API")]
```

Comment:
Is this change related to the `Magic` constant ?

Reply:
It was, but it's an old rule.  The way you get rid of it, is just make the static property readonly.  Allows you to essentially make a static property a constant.  The reason you do this is because constants are basic types and not a reference type like an array.